pages/tutorials/3.0/graphics/sprite.md

```diff
@@ -204,6 +204,21 @@ sf::Sprite loadSprite(const std::filesystem::path& filename)
 } // error: the texture is destroyed here
 ```
 
+Another common way to make this mistake is with containers that can reallocate like `std::vector`.
```

Comment:
I'd either here or below expand "reallocate" to make it clear that it affects all the items and when it usually happens, i.e. when adding a new item.

```suggestion
Another common way to make this mistake is with containers like `std::vector` that can reallocate all the items when inserting an additional item.
```